docs/software/uenv/configure.md

```diff
@@ -58,3 +59,17 @@ The repo is selected according to the following process:
 * if the `--repo` CLI argument is given, use that setting
 * else if `repo` is set in the config file, use that setting
 * else use the default value of `$SCRATCH/.uenv-images`
```

Comment:
given the line you added below, maybe be more generic and don't use SCRATCH? maybe adding a link to the table in `management/#creating-and-using-repositories`

```suggestion
* else use the default system-specific value
```